File: Javascript/1501-1550/1534. Count Good Triplets.js
// https://leetcode.com/problems/count-good-triplets/

var countGoodTriplets = function(arr, a, b, c) {
    let result = 0;

    for(let i=0; i<arr.length; i++){
        for(let j=i+1; j<arr.length; j++){
            for(let k=j+1; k<arr.length; k++){
                if(Math.abs(arr[i] - arr[j]) <= a){
                    if(Math.abs(arr[j] - arr[k]) <= b){
                        if(Math.abs(arr[i] - arr[k]) <= c){
                            result++;
                        }
                    }
                }
            }
        }
    }

    return result;
};

File: Python/1501-1550/1534. Count Good Triplets.py
# https://leetcode.com/problems/count-good-triplets/

class Solution:
    def countGoodTriplets(self, arr: List[int], a: int, b: int, c: int) -> int:
        result = 0

        for i in range(0,len(arr)):
            for j in range(i+1 ,len(arr)):
                for k in range (j+1 , len(arr)):
                    if abs(arr[i] - arr[j]) <= a:
                        if abs(arr[j] - arr[k]) <= b:
                            if abs(arr[i] - arr[k]) <= c:
                                result += 1

        return result
